src/jemalloc.c

```diff
@@ -1459,6 +1475,21 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 				CONF_HANDLE_BOOL(
 				    opt_prof_experimental_use_sys_thread_name,
 				    "prof_experimental_use_sys_thread_name")
+				if (CONF_MATCH("prof_time_resolution")) {
+					if (CONF_MATCH_VALUE("default")) {
+						opt_prof_time_resolution = prof_time_resolution_default;
+					} else if (CONF_MATCH_VALUE("high")) {
+						if (!opt_high_resolution_timer) {
```

Comment:
We should probably also do a runtime-check, since it's possible that the build-time machine has CLOCK_REALTIME support but the run-time machine doesn't (i.e. actually try a gettime call and see if it fails with EINVAL).

Reply:
Just to note that any system that implements a clock_gettime() interface as defined by POSIX.1 has to implement a CLOCK_REALTIME clock_id. A cursory reading of the spec (https://pubs.opengroup.org/onlinepubs/9699919799/functions/clock_getres.html) seems to indicate that CLOCK_REALTIME is probably the only mandatory clock_id that has to be implemented.

Would we expect any system not to implement a minimal posix compaint clock_gettime()?

Reply:
Ah, makes sense to me, I don't think we need a runtime check then.